vue/components/ui/atoms/container/container.vue

```diff
@@ -42,15 +75,35 @@ body.mobile .container-ripe {
 }
 
 .container-ripe > .container-header {
+    align-items: center;
+    display: flex;
     font-size: 0px;
     padding: 24px 24px 20px 24px;
-    text-align: left;
 }
 
 body.tablet .container-ripe > .container-header,
 body.mobile .container-ripe > .container-header {
+    flex-flow: column;
     padding: 20px 15px 20px 15px;
 }
+
+.container-ripe > .container-header > .title {
+    flex: 1;
+}
+
+body.tablet .container-ripe > .container-header > .title,
+body.mobile .container-ripe > .container-header > .title {
+    align-self: flex-start;
+    order: 4;

```

Comment:
Double-check this.

Reply:
See if this can be solved with `-reverse`, even if you have to use it twice (one to change the direction at this level, and the other one to prevent buttons from having a different order)

Reply:
Using `order` seems the right approach. Besides, adding more elements don't affect `.title` order as their default `order` is always 0.

Check this example, where I added 3 elements before and 3 after the buttons (all of this elements are after the title) 
<img width="408" alt="imagem" src="https://user-images.githubusercontent.com/22588915/81921661-dce6b780-95d2-11ea-9cea-8da5bc108628.png">